proposals/0001-template.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Extract WebView From Core
+author:
+- Jamon Holmgren
+date: 2018-07-30
+---
+
+# RFC0000: Extract WebView From Core
+
+## Summary
+
+Extract the current WebView implementation from React Native Core and move it into a third party package.
+
+## Basic example
+
+Currently, WebView is included in React Native core:
+
+```jsx
+import { WebView } from 'react-native';
+```
+
+This would move it to an external package, like so:
+
+```jsx
+import { WebView } from '@infinitered/react-native-webview';
+```
+
+## Motivation
+
+The current WebView implementation is (to my knowledge) not used by Facebook internally. This leads (understandably) to a lack of first-class support for this core API. By moving the component to a third party that is more invested in supporting it, there will be better support.
```

Comment:
Actually, we do use WebView internally, but have not been maintaining it as much.

Reply:
Hello i'm new i made a  webview app using this code

Reply:
import React, { Component } from 'react';
import WebView from 'react-native-webview';

export default class App extends Component {
  render() {
    return (
      <WebView
        source={{ uri: 'http://degrapack.com/radical' }}
        style={{ marginTop: 20 }}
      />
    );
  }
}

Reply:
import React, { Component } from 'react';
import WebView from 'react-native-webview';

export default class App extends Component {
  render() {
    return (
      <WebView
        source={{ uri: 'http://google.com' }}
        style={{ marginTop: 20 }}
      />
    );
  }
}

Reply:
but i want to go back in the webview app and the app closes, can you share the code I Need to fix it